test/Microsoft.ML.Core.Tests/UnitTests/TestEntryPoints.cs

```diff
@@ -1313,7 +1314,7 @@ public void EntryPointMulticlassPipelineEnsemble()
             }
         }
 
-        [ConditionalFact(typeof(BaseTestBaseline), nameof(BaseTestBaseline.LessThanNetCore30OrNotNetCore))]
+        [LessThanNetCore30OrNotNetCoreFact("netcore3.0 output differs from Baseline")]
```

Comment:
`netcore3.0` => `netcoreapp3.0`.  We should use the real TFM here, so we don't confuse people.